public/app/core/components/Login/UserSignup.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+import config from 'app/core/config';
+
+export const UserSignup: FC<any> = ({}) => {
```

Comment:
`FC<void>`

Reply:
Typescript complains about that, so I'm just doing `FC<{}>`